app/scripts/controllers/permissions/index.js

```diff
@@ -244,8 +255,8 @@ export class PermissionsController {
   }
 
   /**
-   * Finalizes a permissions request.
-   * Throws if request validation fails.
+   * Finalizes a permissions request. Throws if request validation fails.
+   * Mutates certain caveats. See function body for details.
```

Comment:
> See function body for details.

This defeats the purpose of the doc comment, no? Could remove this or elaborate on the details here?